spec/requests/subject_and_classses_spec.rb

```diff
@@ -50,9 +50,9 @@
                     'id' => SubAndClass.first.id,
                     'created_at' => SubAndClass.first.created_at.as_json,
                     'subject' => { 'id' => Subject.first.id, 'name' => Subject.first.name.to_s,
-                                   'created_at' => Subject.first.created_at.as_json, 'updated_at' => Subject.first.updated_at.as_json },
+                                   'created_at' => Subject.first.created_at.as_json },
```

Comment:
@josephelias-rubyians  You can add updated_at in the serializer, I think updated_at is something more relevant that created_at.

Reply:
sure